ReactiveCocoa.xcodeproj/xcshareddata/xcschemes/ReactiveCocoa-Mac.xcscheme

```diff
@@ -3,8 +3,8 @@
    LastUpgradeVersion = "0700"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "YES"
-      buildImplicitDependencies = "YES">
+      parallelizeBuildables = "NO"
```

Comment:
Xcode might decide to build `Result` and `ReactiveCocoa` in parallel because there is no target dependency for the former on the latter. This should never happen because `ReactiveCocoa` depends on `Result`. A proper target dependency would require a project reference of the `Result` project on the `ReactiveCocoa` project, which would also add the schemes of the former to the latter.